docs/user_guides/fs/storage_connector/creation/s3.md

```diff
@@ -68,6 +68,12 @@ With this option the [encryption key is managed by AWS KMS](https://docs.aws.ama
 
 If you have SSE-KMS enabled for your bucket, you can find the key ARN in the "Properties" section of the bucket details on AWS.
 
+### Step 5: Add Spark Options (optional)
+Here you can specify any additional spark options that you wish to add to the spark context at runtime. Multiple options can be added as key - value pairs.
+
+!!! tip
+To connect to a non-AWS hosted S3 compliant storage you can add the option with key as `fs.s3a.endpoint` and the endpoint you want to use as value. The storage connector will then be able to read from your specified S3 compatible storage. 
```

Comment:
You need to indent this paragraph for the layout to work correctly.

Reply:
done